### DISCUSSION TITLE
How to change Port Number from 8000

### DISCUSSION THREAD

**Discussion by black-jmyntrn** [outdoorbits/little-backup-box#324] [General]
I thought I asked this once before, cant find the post now, how does one change the port number? It would be nice if this was changeable in the gui

**Comment by outdoorbits** [comment]
I'm travelling now, please have a look at the Apache settings in /etc/apache2/ports

**Reply by outdoorbits** [reply]
... And you have to change the configuration in sites-enabled.

**Reply by black-jmyntrn** [reply]
i dont have /etc/apache2/ports its a basic raspberry pi os install

**Comment by outdoorbits** [comment]
You must have, it's the Linux configuration of the webserver.

**Reply by black-jmyntrn** [reply]
so.. im trying to install LBB with Companion PI... just did a new install and only installed companion as I was able to change the port... now I dont have LBB installed as everytime I try to run your script i get 
```

-bash: install-error.log: Permission denied
curl: (23) Failure writing output to destination
```

so now.. im going to redo it all and install LBB first and change the port then see if I can get companion pi installed.

**Reply by black-jmyntrn** [reply]
ahhh /etc/apache2/ports.conf  :)
 and the below file... just hit reboot and crossing fingers!
/etc/apache2/sites-enabled/little-backup-box.conf
